apps/intellij/src/main/kotlin/dev/nx/console/generate_ui/run_generator/run_task.kt

```diff
@@ -1,22 +1,28 @@
 package dev.nx.console.generate_ui.run_generator
 
+import com.intellij.execution.configurations.GeneralCommandLine
+import com.intellij.execution.executors.DefaultRunExecutor
+import com.intellij.execution.filters.TextConsoleBuilderFactory
+import com.intellij.execution.process.OSProcessHandler
+import com.intellij.execution.process.ProcessHandler
+import com.intellij.execution.ui.RunContentDescriptor
+import com.intellij.execution.ui.RunContentManager
 import com.intellij.javascript.nodejs.interpreter.NodeJsInterpreterManager
 import com.intellij.javascript.nodejs.npm.NpmUtil
 import com.intellij.openapi.application.ApplicationManager
 import com.intellij.openapi.application.ModalityState
 import com.intellij.openapi.diagnostic.logger
 import com.intellij.openapi.project.Project
-import com.intellij.openapi.wm.ToolWindowManager
+import dev.nx.console.NxIcons
 import io.ktor.utils.io.errors.*
-import org.jetbrains.plugins.terminal.ShellTerminalWidget
-import org.jetbrains.plugins.terminal.TerminalToolWindowFactory
-import org.jetbrains.plugins.terminal.TerminalView
 
-fun runGenerator(generator: String, flags: List<String>, project: Project): Unit {
+fun runGenerator(
```

Comment:
This doesnt look like it's being used anymore. We can probably delete the whole file now.